tools/spi-hdlc-adapter/spi-hdlc-adapter.c

```diff
@@ -1722,7 +1713,12 @@ int main(int argc, char *argv[])
     argc -= optind;
     argv += optind;
 
-    if (argc >= 1)
+    if (argc == 0)
```

Comment:
prefer not change `argc` and `argv`, just compare `argc` and `optind` is good.

Reply:
Updated.